### PR TITLE
Append cert to new certpath

### PR DESCRIPTION
**Pull request by joshsamuelson**

### CODE REVIEW

File: manifests/cacert.pp [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
       ensure => file,
       source => "${classroom::confdir}/ssl/certs/ca.pem",
       notify => Exec['trust classroom ca'],
     }
 
-    exec { 'trust classroom ca':
-      command     => '/usr/bin/update-ca-trust extract',
-      onlyif      => '/usr/bin/update-ca-trust enable',
-      refreshonly => true,
+    if versioncmp($::aio_agent_verstion, '1.3.2') >= 0 {
```
Comment (binford2k) [inline manifests/cacert.pp]: typo

**Comment by binford2k** [comment]
This has no other side effects?

**Comment by joshsamuelson** [comment]
I doubt it has any other side effects and I didn't see any.  It's just appending a cert to a file of certs.

**Comment by binford2k** [comment]
right, but is it one of the managed certificates? Do subsequent Puppet runs overwrite it, or does it get copied around, etc?

**Comment by joshsamuelson** [comment]
I just double checked and puppet runs don't overwrite it.  It seemed pretty stable when I was running through the labs.

**Comment by binford2k** [comment]
sweet

**Pull request merged**